docs/core/docker/building-net-docker-images.md

```diff
@@ -2,224 +2,251 @@
 title: Building .NET Core Docker Images
 description: Understanding Docker images and .NET Core
 keywords: .NET, .NET Core, Docker
-author: spboyer
-ms.author: shboyer
-ms.date: 09/06/2017
+author: johalex
```

Comment:
I'll try to review your PR shortly. your GH alias is JRAlexander

Reply:
Thanks! Good catch